scripts/common.sh

```diff
@@ -0,0 +1,11 @@
+if [ -t 1 ] ; then
+  RED='\033[0;31m'
+  GREEN='\033[0;32m'
+  BLUE='\033[0;34m'
+  RS='\033[0m' # reset to no color
+else
+  RED=''
+  GREEN=''
```

Comment:
could also put the literal `RED` then we can use search to show all errors or whatever when looking at the logs